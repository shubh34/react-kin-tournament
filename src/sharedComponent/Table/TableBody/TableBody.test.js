import React from 'react';
import { shallow } from 'enzyme';
import TableBody from './TableBody';

describe('TableBody', () => {
	it('renders without crashing', () => {
		const wrapper = shallow(<TableBody>TableBody</TableBody>);
		expect(wrapper).toMatchSnapshot();
	});
});
