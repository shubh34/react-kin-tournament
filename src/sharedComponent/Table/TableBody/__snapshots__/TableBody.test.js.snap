// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableBody renders without crashing 1`] = `
<tbody>
  TableBody
</tbody>
`;
